invenio_oauthclient/views/client.py

```diff
@@ -271,3 +271,19 @@ def rest_disconnect(remote_app):
         return _disconnect(remote_app)
     except OAuthRemoteNotFound:
         abort(404)
+
+
+@blueprint.route("/logout")
+def logout():
+    """Client logout view.
+
+    This URL should be called by setting `SECURITY_POST_LOGOUT_VIEW = /oauth/logout`
+    """
+    remote_name = session.pop("remote_name")
```

Comment:
I am wondering if it could be possible that this is non-existing... do you think it might makes sense to check that?
```
remote_name = session.pop("remote_name", None)
if remote_name:
    ...
```